python/py-twine/Portfile

```diff
@@ -40,6 +41,7 @@ if {${name} ne ${subport}} {
     depends_build-append \
         port:py${python.version}-setuptools
```

Comment:
while you're making changes: the port installs a `console_script` and, therefore, requires `setuptools` also at runtime. Can you please change the `depends_build-append` here to `depends_lib-append`?

Reply:
Just making sure you didn't mean `depends_run-append`?

Reply:
no I didn't. As you can see from `setup.py` it is also needed at build time. If you set it to `depends_run-append` it will *only* be available at runtime; setting it to `depends_lib` does make sure it's there at build- and run-time.

Reply:
K it's done

Reply:
If it is only set as a build dependency, it won't get installed if the binary is installed. If it is also needed at run-time, it should be set as a library dependency (`depends_lib`).